README.md

```diff
@@ -168,7 +167,7 @@ shell with:
 python x.py build
 ```
 
-Currently, building Rust only works with some known versions of Visual Studio. If
+Right now, building Rust only works with some known versions of Visual Studio. If
```

Comment:
why this change?

Reply:
its not necessary, but i find "Right now" simpler than "Currently"

Reply:
I think "currently" sounds better, "right now" sounds too much like something very temporary that might change at any second too me, but this has been a problem for quite some time I think

Reply:
i think its the opposite. "i'm not currently dating anyone" but you might in the future. "my dog isn't currently house-trained" but it might be soon.

right now just means at this present moment. it doesn't assume anything about the duration.

what do you think?